front/app/modules/commercial/matomo/index.test.ts

```diff
@@ -28,23 +28,23 @@ describe('matomo', () => {
 
   // Note there is duplication in the values coming back from the array
   // but Matomo deals with this and does not make duplicate requests
-  it('sets up matomo and makes call for initial page after consent is given', () => {
-    (config as any).beforeMountApplication();
-    eventEmitter.emit<any>('destinationConsentChanged', { matomo: true });
```

Comment:
Will re-enable these tests when I start working on fixing the duplicate calls ticket